cmd/docker/docker.go

```diff
@@ -260,6 +276,9 @@ func isSupported(cmd *cobra.Command, details versionDetails) error {
 			if _, ok := f.Annotations["experimental"]; ok && !hasExperimental {
 				errs = append(errs, fmt.Sprintf("\"--%s\" is only supported on a Docker daemon with experimental features enabled", f.Name))
 			}
+			if _, ok := f.Annotations["experimentalCli"]; ok && !hasExperimentalCli {
+				errs = append(errs, fmt.Sprintf("\"--%s\" is only supported on a Docker cli with experimental features enabled", f.Name))
```

Comment:
"only supported when experimental cli features are enabled"

I just realized all of this was duplicated. We should be able to declare this visitor once, and call it for the root command case. Doesn't need to be part of this PR.

Reply:
@dnephin yep, we realized that too in #721 just didn't want to add more code to it 😛